### PR TITLE
Model load with gcs fuse benchmarking tool

### PR DESCRIPTION
**Pull request by coolkp**
Benchmarking tool for benchmarking data loading with gcsfuse

### CODE REVIEW

File: tools/model-load-benchmark/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: pod-creator-binding
+subjects:
+- kind: User
+  name: gke_kunjanp-gke-dev-2_us-west4_gpu-dev-cluster 
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin  
+  apiGroup: rbac.authorization.k8s.io
```
Comment (genlu2011) [inline tools/model-load-benchmark/rbac.yaml:12]: nit: newline is missing in multiple files in this PR.

File: tools/model-load-benchmark/example-pod.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    app: Kunjan
```
Comment (genlu2011) [inline tools/model-load-benchmark/example-pod.yaml:5]: remove this?

File: tools/model-load-benchmark/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+Here’s an expanded README with a section detailing how to install the Go package as a CLI:
+
+---
+
+# Benchmarker CLI
+
+A CLI tool for configuring and running benchmarks on various configurations.
```
Comment (genlu2011) [inline tools/model-load-benchmark/README.md:7]: Could you provide more context about this tool from the user’s perspective? For instance, the tool is designed to help users identify optimal GCSFuse configurations that enhance model loading times, ensuring smoother and more efficient operations.

File: tools/model-load-benchmark/README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+## Table of Contents
```
Comment (genlu2011) [inline tools/model-load-benchmark/README.md:9]: What do you think of adding a simple explanation of the architecture?

File: tools/model-load-benchmark/README.md (hunk 3 above)
Comment (coolkp) [reply tools/model-load-benchmark/README.md]: Great suggestion! Resolved, thanks i had forgotten to change the background

File: tools/model-load-benchmark/README.md (hunk 4 above)
Comment (coolkp) [reply tools/model-load-benchmark/README.md:4]: Good idea, added.

File: tools/model-load-benchmark/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+matplotlib
+pyyaml
```
Comment (genlu2011) [inline tools/model-load-benchmark/requirements.txt:2]: nit: newline here?

Comment (coolkp) [reply tools/model-load-benchmark/requirements.txt]: done

**Review by genlu2011 — APPROVED** [review]
Thanks for your contribution!

**Comment by achandrasekar** [comment]
@coolkp can we add this under our benchmarks directory where we have other benchmarks like dlio, lpg and locust? It also gives us the ability to use the terraform modules for cluster provisioning if needed.

**Review by achandrasekar — CHANGES_REQUESTED** [review]
@coolkp can we add this under our benchmarks directory where we have other benchmarks like dlio, lpg and locust? It also gives us the ability to use the terraform modules for cluster provisioning if needed.

**Review by coolkp — COMMENTED** [review]
sounds good

**Review by achandrasekar — APPROVED** [review]
Thanks for making the change!

**Pull request merged**